src/app/routes/cpsAsset/getInitialData/insertPodcastPromo/index.js

```diff
@@ -5,37 +5,56 @@ import insert from 'ramda/src/insert';
 import slice from 'ramda/src/slice';
 import findNthIndex from '#lib/utilities/findNthIndex';
 
+// Podcast promos are inserted if the page content has a continuous sequence of paragraph blocks
+// These paragraph blocks must have a minimum character count in total to ensure there is
+// enough text to wrap around the promo component
+const MINIMUM_CHARACTER_COUNT = 940;
```

Comment:
Does this minimum number take into account character sizing for the different fonts and sizing of characters in the services this is being used on?

Reply:
It's a pretty rough metric - it's hard to get this number right considering different font sizes and (especially) different screen sizes.  The 940 number came from @JonBeeb - I believe it's a rough average based on the services we're likely to be running this experiment on